### DISCUSSION TITLE
SEB failed to start a new session

### DISCUSSION THREAD

**Discussion by YoniA** [SafeExamBrowser/seb-win-refactoring#165] [Q&A] [answered]
Hello,

I have installed SEB 3.1.1 on several machines and can't start the application. 

![image](https://user-images.githubusercontent.com/6865423/130353778-55acf7c7-dcce-4394-80d4-4f2f93bf2963.png)

I have inspected the logfiles (attached here), and the first failure seems to be:
`ERROR: Failed to start new service session within 30 seconds!`

Rebooting the machines or even reinstalling SEB does not solve the problem.

Looking up the SourceForge help forum and here, I could not find a post with a logfile containing a similar line.

How can I solve this problem?

Thanks in advance!

[2021-08-18_13h15m53s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7027262/2021-08-18_13h15m53s_Runtime.log)
[2021-08-18_13h15m53s_Service.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7027263/2021-08-18_13h15m53s_Service.log)

**Comment by dbuechel** [comment, marked as answer]
Which Windows 10 version are you using (e.g. "Windows 10 Pro 20H2")?

**Reply by dbuechel** [reply]
SEB fails to stop the Windows Update service, that's why starting a new session fails: `Service wuauserv was not found on computer`

**Reply by YoniA** [reply]
I am using Windows 10 Education, Version 1903.

So what can I do about the failure to stop the update service issue?

**Reply by dbuechel** [reply]
First, we suggest to always keep your operating system up to date, i.e. install all the latest Windows updates. Secondly, could you please check whether there is a service called "wuauserv" (Windows Update) on your system? You can do so by opening the task manager and searching the list in tab "Services".

**Reply by dbuechel** [reply]
Or, if you have control over the SEB configuration you're using, you can simply bypass the service functionality of SEB by checking "Ignore SEB Service" under tab "Security" in the configuration tool.

**Reply by YoniA** [reply]
Thanks for your help. The system is configured to stop this service on every restart. So when SEB is invoked, this service is already stopped.

Maybe you could add a patch to the code, that checks if the service is already stopped (no need to kill it, or crash).

P.S, I do have control over the configuration file. However, my exam is quite secure. What are the implications if I configure it to ignore SEB service?

**Reply by dbuechel** [reply]
That's okay if it is already stopped. The problem was that apparently the service could not be found at all on the system. We generally recommend ignoring the SEB service, the exam still should be sufficiently secure without it. The users will then be able to e.g. perform actions on the Windows Security Screen (invoked by CTRL+ALT+DEL), but SEB detects at least some of them and renders a lock screen when returning to the exam.

**Reply by YoniA** [reply]
O.K, will do that.

Thanks for your kind help.
